### PR TITLE
added py docu for shape/constraints, removed some typos

### PR DESCRIPTION
**Pull request by richter-t**

### CODE REVIEW

File: doc/ug/part.tex [hunk 1]
```diff
@@ -1127,6 +1127,106 @@ \section{\texttt{constraint}: Setting up constraints}\label{sec:constraint}
...
+			outer_radius = OR, inner_radius = IR, width = W, opening_angle = a,
+			direction = D)
+		\end{pycode}
+		creates a hollow cone whose axis is aligned to \codebox{(a_x, a_y, a_z)}
+		located at position \codebox{(x, y, z)}, which inner and outer radii are
+		\codebox{IR} and \codebox{OR}, respectively. The width and opening angle are
+		given by \codebox{W} and \codebox{a}.
+\end{enumerate}
+
+The direction paramter for the shapes specifies wheter it will act torwards the
```
Comment (KaiSzuttor) [inline doc/ug/part.tex]: typo

File: doc/ug/part.tex [hunk 2]
```diff
@@ -1127,6 +1127,106 @@ \section{\texttt{constraint}: Setting up constraints}\label{sec:constraint}
...
+outside \codebox{D=1} or inside \codebox{D=-1}.
+All those shapes can be used as constraints and added to the systems constraints by
+passing a initialized shape object to \codebox{system.constraints}
```
Comment (KaiSzuttor) [inline doc/ug/part.tex]: an

**Comment by rbardak** [comment]
@RudolfWeeber shapes and constraints are used in the tutorials.

**Comment by RudolfWeeber** [comment]
On Fri, Oct 07, 2016 at 12:57:31PM -0700, Robin Bardakcioglu wrote:
> @RudolfWeeber shapes and constraints are used in the tutorials.
I merged this pr into the python branch, already. Is sth missing?

**Comment by rbardak** [comment]
> I merged this pr into the python branch, already. Is sth missing?

I edited my original message, apparently there is no email notification fo that. I did not find the documentation in the beginning so I thought you didn't merge it. I found it in the branch after I nagged.

**Pull request closed without merging**